packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name.jsx

```diff
@@ -81,20 +81,20 @@ const MyProfileName = () => {
                             </DesktopWrapper>
                             {rating_average ? (
                                 <React.Fragment>
-                                    <div className='my-profile-name__rating__row'>
+                                    <div className='my-profile-name--rating__row'>
```

Comment:
should this be 'my-profile-name__rating--row' or 'my-profile-name--rating__row'?

Reply:
ohh ya it should be `my-profile-name__rating__row`